src/Providers/IORouteServiceProvider.php

```diff
@@ -368,6 +368,10 @@ public function map(Router $router, ApiRouter $api)
 
             $router->get('a-{itemId}', 'IO\Controllers\ItemController@showItemFromAdmin')
                 ->where('itemId', '[0-9]+');
+            
+            $router->get('{slug}_t{tagId}', 'IO\Controllers\TagController@showItemByTag')
+            ->where('slug', '.*')
```

Comment:
s.o. Evtl. kann `slug` eingegrenzt werden, um `"/"` nicht zu erlauben. in dem Fall könnte man den Platzhalter noch in `tagName` umbenennen

Reply:
mit welchem regex kann ich / verbieten ?
so ? ```[^\/]*```

Reply:
müsste gehen. Am besten einmal testen